src/WorldState/WorldStateEndpoints.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace WorldState
+{
+    internal static class WorldStateEndpoints
+    {
+        public const string Base         = "https://api.warframestat.us";
+        public const string Alerts       = "/alerts";
+        public const string Acolytes     = "/persistentEnemies";
+        public const string CetusCycle   = "/cetusCycle";
+        public const string Conclaves    = "/conclaveChallenges";
+        public const string Construction = "/constructionProgress";
```

Comment:
you're probably gonna want to reference https://docs.warframestat.us if you're attempting to document all of the endpoints

Reply:
For now I'm using this file as a resource file to collect all string literals so they can be corrected or changed in a central place if typo or updated. I'll add docs after I make sure everything works.

On a side note, [the doc page](https://docs.warframestat.us/) looks like it has a weird color scheme in my browser, not sure if just me. I can barely make out any documentation texts. And I'm sure I don't have fancy HDR or Dark Mode stuff turned on to make everything look bad.

<img width="947" alt="Screenshot 2019-11-20 173040" src="https://user-images.githubusercontent.com/3415065/69226897-ebbc5680-0bbb-11ea-8648-56c95df01e8d.png">

Reply:
yes, it is a weird color right now. i need to do some more manual styling on it.